collections.js

```diff
@@ -118,8 +118,36 @@ LinkedList.prototype.find = function(value) {
     node = node.next;
   }
   return null;
+};
+
+//------------------------------------------------------------------------------
+// Simple Queue.
+
+function SimpleQueue() {
```

Comment:
I would just call this Queue. It's more consistent with other languages (C++, Java, C#, etc.)